tests/integration_tests/partial_index/run.sh

```
@@ -0,0 +1,70 @@
+
+	run_sql "show create table partial_index.t2;" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT} &&
+		check_contains "idx_partial_create_index" &&
+		check_contains "\`col2\` > 100"
+
+	run_sql "show create table partial_index.t3;" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT} &&
+		check_contains "idx_partial_alter" &&
+		check_contains "\`col2\` > 100"
+}
+
+trap 'stop_test $WORK_DIR' EXIT
+run $*
```

Shellcheck (0.11.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)